### ISSUE TITLE
support jest 27

### ISSUE THREAD

**Issue by Tallyb** [ionic-team/stencil#2918]
**Stencil version:**

```
 @stencil/core@2.5.2
```

**I'm submitting a:**

[ ] bug report
[X] feature request
[ ] support request => Please do not submit support requests here, use one of these channels: https://stencil-worldwide.herokuapp.com/ or https://forum.ionicframework.com/

**Current behavior:**
Tests are working with jest 26. 
You cannot upgrade to jest 27 since you get jasmine is not defined. 
Setting up   `testRunner: 'jest-jasmine2'` in the configuration to retain old jasmine does not help either and throws Cannot read property 'rootDir' of undefined. 

**Expected behavior:**
For the upcoming stencil 2.6, would it be possible to support jest 27? 

**Steps to reproduce:**
Upgrade `jest` and `jest-cli` to 27. Try running tests.

**Comment by antoniosZ** [comment]
I get
```
runJest: Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: Package subpath './build/cli/args' is not defined by
           "exports" in .../node_modules/jest-cli/package.json
```

**Comment by splitinfinities** [comment]
Hi Tally! We're working on this now, just wanted to let you know! When we have next steps, we'll loop you in. Thank you so much for the patience!

**Comment by rwaskiewicz** [comment]
Hey ionitron-bot, we didn't mean to close this. Bad bot!

**Comment by rwaskiewicz** [comment]
https://github.com/ionic-team/stencil/pull/3189 includes support for Jest 27, and will be included in the next minor release of Stencil (v2.13.0)

**Comment by rwaskiewicz** [comment]
Hey folks, this should be fixed in [v2.13.0](https://github.com/ionic-team/stencil/releases/tag/v2.13.0) of Stencil, which was just released with Jest 27 support. I'm going to close this issue for now, but if this comes up again, please feel free to file a new ticket!
